pkgs/os-specific/linux/bpftrace/default.nix

```diff
@@ -58,12 +73,16 @@ stdenv.mkDerivation rec {
   cmakeFlags =
     [ "-DBUILD_TESTING=FALSE"
       "-DLIBBCC_INCLUDE_DIRS=${bcc}/include"
+      "-DINSTALL_EXAMPLES=OFF"
     ];
 
-  # nuke the example/reference output .txt files, for the included tools,
-  # stuffed inside $out. we don't need them at all.
+  # Pull BPF scripts into $PATH (next to their bcc program equivalents), but do
+  # not move them to keep `${pkgs.bpftrace}/share/bpftrace/tools/...` working.
```

Comment:
We don't have anything using these paths in nixpkgs.
Users could depend on it in their repos/scripts but I think it's fine to just move them with a note in next releases notes -- the tools are mostly here for interactive use and not really thought out for scripting in the first place so the impact should be minimal... hopefully? Better than having to keep the links forever anyway.

Reply:
> We don't have anything using these paths in nixpkgs.

If it were just for cases like this, we could obviously move the files and fix all consumers...

> Users could depend on it in their repos/scripts but I think it's fine to just move them with a note in next releases notes -- the tools are mostly here for interactive use and not really thought out for scripting in the first place so the impact should be minimal... hopefully? Better than having to keep the links forever anyway.

This sounds like a viable plan as soon as we sorted out a final solution with upstream.

Until then I'd like to symlink and reduce friction for nixpkgs users (which is too unstable enough already).
This wouldn't be forever.